Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json

```diff
@@ -7599,7 +7962,22 @@
 															"});\r",
 															""
 														],
-														"type": "text/javascript"
+														"type": "text/javascript",
+														"packages": {}
+													}
+												},
+												{
+													"listen": "prerequest",
+													"script": {
+														"exec": [
+															"// Only run if Composites are available\r",
```

Comment:
This change adds a guard condition on certain tests that assume Composites is enabled (in a non-Composites test fixture).